app/controllers/api/v2/hosts_controller.rb

```diff
@@ -265,8 +265,12 @@ def facts
 
       api :PUT, "/hosts/:id/rebuild_config", N_("Rebuild orchestration config")
       param :id, :identifier_dottable, :required => true
+      param :only, Array, :desc => N_(<<-eos
+Limit rebuild steps, valid steps are #{(Nic::Managed.rebuild_methods.values + Host::Managed.rebuild_methods.values + Nic::Managed.rebuild_methods.keys + Host::Managed.rebuild_methods.keys).join(', ')}
+eos
+)
```

Comment:
@lzap I tried a constant in Host::Managed but kept getting undefined method errors on rebuild_methods.  Using a class method seems to work